trulens_eval/examples/expositional/frameworks/nemoguardrails/iterating_guardrails_apps.ipynb

```diff
@@ -0,0 +1,1250 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.          "text1":"{question_lens}",\
</pre><p>The nemo persons suggested that using the variables version of the lenses here (for example <code>{context_lens}</code> ) is a bit confusing. Because of this, there is the other notebook that does it a bit simpler by defining python methods for each of these flows. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/1131/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>